### PR TITLE
ghcid: add page

### PR DESCRIPTION
**Pull request by sohang3112**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):** 0.8.9

### CODE REVIEW

**Comment by sohang3112** [comment]
@sebastiaanspeck @vitorhcl Please review this PR

**Comment by sohang3112** [comment]
@sebastiaanspeck I pulled upstream changed into my branch, please check PR now.

File: pages/common/ghcid.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+> Continuously displays compile errors, warnings, and test results.
+> More information: <https://github.com/ndmitchell/ghcid>.
+
+- Start `ghcid` and monitor a Haskell file for changes:
+
+`ghcid {{path/to/Main.hs}}`
+
+- Start `ghcid` with a specific command, such as loading a Stack or Cabal project:
+
+`ghcid --command "stack ghci {{path/to/Main.hs}}"`
```
Comment (spageektti) [inline pages/common/ghcid.md:13]:
```suggestion
`ghcid --command "{{stack ghci Main.hs}}"`
```
In this case, the whole command is a placeholder.

File: pages/common/ghcid.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+`ghcid --run={{action}} {{path/to/Main.hs}}`
+
+- Set maximum height and width (default to console height and width):
+
+`ghcid --height={{height}} --width={{width}} {{path/to/Main.hs}}`
+
+- Write full GHC compiler output to a file:
+
+`ghcid --outputfile={{path/to/outputfile.txt}} {{path/to/Main.hs}}`
```
Comment (spageektti) [inline pages/common/ghcid.md:25]:
```suggestion
`ghcid --outputfile={{path/to/output_file.txt}} {{path/to/Main.hs}}`
```

**Review by gutjuri — APPROVED** [review]
LGTM after the suggestions

**Comment by sohang3112** [comment]
@gutjiri Committed your suggestions - please check now

**Comment by sohang3112** [comment]
@spageektti Ok I'll keep it in mind from next time. Please review PR after changes when you're free

**Review by spageektti — APPROVED** [review]

**Pull request merged**